### ISSUE TITLE
add `activeEditorState` window title variable to editor's a11y help dialog

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#240393]
related #239887 

cc @jooyoungseo and @rperez030 while we decide if this should be the default, I'm going to add it to the editor's help dialog for discoverability.
